### ISSUE TITLE
Can't activate with anything other than the first pool that comes back from zpool list

### ISSUE THREAD

**Issue by bhechinger** [cneira/zcage#31]
I have two pools. `rpool` and `tank`. I want to activate zcage to use the pool `tank` for all the zones it creates.

It refuses to allow me to do this. It looks like GetPool() only ever returns the first one no matter what, which isn't idea.

Also it's now telling me (I was on 1.4.6, messing with master currently) that it's already activated, even though it isn't because it's checking for things I'm pretty sure it shouldn't be touching:

```
function Isactivated(out) {
    if (fs.existsSync(ZCAGE.DS) && 
	fs.existsSync(ZCAGE.LXBRAND) &&
	fs.existsSync(ZCAGE.BHYVEBRAND)) {
```

Please help, this is an awesome piece of software otherwise!

**Comment by cneira** [comment]
@bhechinger  
I'm sorry for the late response, I'll take a look. Thanks for letting me know.

**Comment by cneira** [comment]
@bhechinger  this has been fixed by 39c907cfece55acb78da86245b169f7737d216e2

You could specify alternative pools to provision containers.

Re-open the issue if needed.

**Comment by bhechinger** [comment]
Perfect, thanks!
